server/database/queries/addCourse.js

```diff
@@ -0,0 +1,12 @@
+const connection = require('../config/db_connection');
+
+const insertCourse = (userInfo) => {
+  const queryValues = Object.keys(userInfo).map(info => userInfo[info]);
```

Comment:
@AbdallahAmmar96 I wonder, does this query still work if you replace the keys().map() line with:
`const queryValues = Object.values(userInfo)`? If so, that saves a little bit of code 🤷‍♂️
Though it might not give the order of values that the query expects.

Reply:
@tomduggan85  yes, it's still working .. 
I replace it